docs/cloud/azureai/manage-flows.md

```diff
@@ -102,6 +98,15 @@ flow = pf.flows.create_or_update(
 
 ::::
 
+On Azure portal, you can see the created flow in the flow list.
+
+![img](../../media/cloud/manage-flows/flow_create_1.png)
+
+And the flow source folder on file share is `Users/<alias>/promptflow/<flow-display-name>`:
+
+![img](../../media/cloud/manage-flows/flow_create_2.png)
+
+
```

Comment:
We could add a note under this section stating, 'If not specified, the display name will default to the flow folder name + timestamp.
![image](https://github.com/microsoft/promptflow/assets/32232238/1586ff6c-07b7-4b2d-bf21-cc03630ae159)

Reply:
Good point, I will open another pr to improve the doc according to your suggestions.